### ISSUE TITLE
Unable to load vector drawable resources with app:srcCompat="@drawable/my_drawable" from Android Library project

### ISSUE THREAD

**Issue by ggirard07** [xamarin/AndroidSupportComponents#8]
### Xamarin.Android Version (eg: 6.0):
Xamarin.Android   6.1.1.1 (7db2aac)

### Operating System & Version (eg: Mac OSX 10.11):
Microsoft Visual Studio Enterprise 2015
Version 14.0.25123.00 Update 2
Microsoft .NET Framework
Version 4.6.01055

### Support Libraries Version (eg: 23.3.0):
23.4.0

### Describe your Issue:
On Android KitKat platforms and above, activities from an Android **library** project using vector drawables with AppCompat are unable to resolve drawable resources, resulting in an unhandled exception crashing the app.

A simple workaround is to manually add the `<AndroidResgenExtraArgs>"--no-version-vectors"</AndroidResgenExtraArgs>` to the Android **application** project file.

### Steps to Reproduce (with link to sample solution if possible):
A reproduction sample is available here https://github.com/ggirard07/AppCompatRepro
Or you could do it by yourself by following the steps:
1. Create a new Android Application project (blank app)
2. Add a new Android Library project to the solution
3. Implement an activity in the Library project that consume vector drawables with AppCompat and srcCompat attribute.
4. Modify the Android application project to start the Android library project activity.

### Include any relevant Exception Stack traces, build logs, adb logs:
```
07-04 12:50:01.359 E/mono-rt (13471): android.content.res.Resources$NotFoundException: File res/drawable/ic_flashlight.xml from drawable resource ID #0x7f02004b
07-04 12:50:01.359 E/mono-rt (13471): 	at android.content.res.Resources.loadDrawable(Resources.java:2235)
07-04 12:50:01.359 E/mono-rt (13471): 	at android.content.res.Resources.getDrawable(Resources.java:733)
07-04 12:50:01.359 E/mono-rt (13471): 	at android.content.res.Resources.getDrawable(Resources.java:711)
07-04 12:50:01.359 E/mono-rt (13471): 	at android.support.v7.widget.ResourcesWrapper.getDrawable(ResourcesWrapper.java:12
```

**Comment by kjeremy** [comment]
I'm seeing the exact same problem including a vector drawable in some menu.xml.  This used to work fine in 23.3 but fails in 23.4.  The workaround above fixed the problem.  This nuget package needs to add the MSBuild argument.

See: https://medium.com/@chrisbanes/appcompat-v23-2-age-of-the-vectors-91cbafa87c88#.fybg1lxa7

**Comment by Redth** [comment]
It looks like there is a .targets file missing in the nuget for Vector Drawable 23.4 which is supposed to take care of adding this for you.

While it's still in the .nuspec: https://github.com/xamarin/AndroidSupportComponents/blob/master/vector-drawable/nuget/Xamarin.Android.Support.Vector.Drawable.nuspec#L21-L22  It looks like there was a packaging issue with that version which caused the assembly and targets file to be omitted.

We will fix this in v23.4.0.1

**Comment by Redth** [comment]
Fixed back in v23.4.0.1
